source/scss/_pf-basic-tabs.scss

```diff
@@ -0,0 +1,52 @@
+//
+// Tabs
+//
+
+.nav-link {
+    color: $pf-tabs-color;
+}
+
+.nav-tabs {
+
+  // overrides bootstrap's spacing
+  .nav-link {
+    padding: $pf-spacer-xxxs $pf-spacer-sm;
+  }
+  // overrides bootstrap to make the tabs butt next to each other
+  .nav-item + .nav-item {
```

Comment:
My intention with nesting everything is to clear understand in a visual way when we are adding more specificity.

What do you think? worth enforcing it or just a waste of characters?